### DISCUSSION TITLE
devices.js how to update

### DISCUSSION THREAD

**Discussion by GekoCH** [Koenkk/zigbee2mqtt#5275] [Devices]
I saw that my device would be supported by the zigbee-herdsman-converters devices.js.
However the current devices.js in my zigbee2mqtt folder is older and even after the update process described here (https://www.zigbee2mqtt.io/getting_started/running_zigbee2mqtt.html) I still have the old devices.js.
How can I update this?

**Comment by wb441** [comment]
Are you running zigbee2mqtt in a docker container or did you install it directly on your host system?
Anyway, you could try removing the file and putting the new one there instead.
A reboot will also help :)

**Comment by GekoCH** [comment]
Yes I downloaded it directly form the zigbee-herdsman-converters github page but why does it not automatically do that? I've installed it locally not in a docker

**Reply by wb441** [reply]
It could be for many reasons. The best shot you have, is to remove the file from the location it's in, and copy in the one from github. Can you report back how it went?

**Reply by GekoCH** [reply]
That's what I've already done before posting. I was just curious why this file is not up to date.

**Reply by wb441** [reply]
So the contents are not as you expect? I assume that what you can see on github is what you get when you download the development branch.
So there could indeed be a difference in what you download in the 'release' version vs the 'dev' version.

With the version that you downloaded, is your device now working?

**Reply by GekoCH** [reply]
Yep it is working, but without searching in the original file on the zigbee-herdsman-converters github page it would not be working
